### ISSUE TITLE
Migration error from sqlite3 to postgres

### ISSUE THREAD

**Issue by jlixfeld** [respawner/peering-manager#84]
### Environment
* Python version: 3.5.3
* Peering Manager version:  master

### Steps to Reproduce

```
root@peeringmanager:/opt/peering-manager# ./scripts/migrate-from-sqlite.sh
```

### Expected Behavior

The command should have run successfully and imported `datadump.json` from sqlite3 to postgres.

### Observed Behavior

```
root@peeringmanager:/opt/peering-manager# ./scripts/migrate-from-sqlite.sh      
Please upgrade and change your configuration.py file to use PostgreSQL
Press enter when you are ready
Migrating database (python3 manage.py migrate --run-syncdb)...
Operations to perform:
  Synchronize unmigrated apps: django_filters, django_tables2, messages, staticfiles
  Apply all migrations: admin, auth, contenttypes, peering, peeringdb, sessions, utils
Synchronizing apps without migrations:
  Creating tables...
    Running deferred SQL...
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying auth.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying peering.0001_initial... OK
  Applying peering.0002_auto_20170820_1809... OK
  Applying peering.0003_auto_20170903_1235... OK
  Applying peering.0004_auto_20171004_2323... OK
  Applying peering.0005_auto_20171014_1427... OK
  Applying peering.0006_auto_20171017_1917... OK
  Applying peering.0007_auto_20171202_1900... OK
  Applying peering.0008_auto_20171212_2251... OK
  Applying peering.0009_auto_20171226_1550... OK
  Applying peering.0010_auto_20171228_0158... OK
  Applying peering.0011_auto_20180329_2146... OK
  Applying peering.0012_auto_20180502_1733... OK
  Applying peering.0013_auto_20180505_1545... OK
  Applying peering.0014_auto_20180519_2128... OK
  Applying peering.0015_peeringsession_password... OK
  Applying peering.0016_auto_20180726_1307... OK
  Applying peering.0017_auto_20180802_2309... OK
  Applying peering.0018_auto_20181014_1612... OK
  Applying peering.0019_router_netbox_device_id... OK
  Applying peering.0020_auto_20181105_0850... OK
  Applying peering.0021_auto_20181113_2136... OK
  Applying peering.0022_auto_20181116_2226... OK
  Applying peering.0023_auto_20181208_2202... OK
  Applying peering.0024_auto_20181212_2106... OK
  Applying peering.0025_auto_20181212_2322... OK
  Applying peering.0026_autonomoussystem_potential_internet_exchange_peering_sessions... OK
  Applying peering.0027_auto_20190105_1600... OK
  Applying peering.0028_internetexchangepeeringsession_is_router_server... OK
  Applying peering.0029_auto_20190114_2141... OK
  Applying peering.0030_directpeeringsession_router... OK
  Applying peeringdb.0001_initial... OK
  Applying peeringdb.0002_auto_20180101_0203... OK
  Applying peeringdb.0003_networkixlan_name... OK
  Applying peeringdb.0004_auto_20180121_2207... OK
  Applying peeringdb.0005_auto_20180122_1845... OK
  Applying peeringdb.0006_auto_20180213_0031... OK
  Applying peeringdb.0007_peerrecord... OK
  Applying peeringdb.0008_auto_20181212_2106... OK
  Applying peeringdb.0009_auto_20181212_2322... OK
  Applying sessions.0001_initial... OK
  Applying utils.0001_initial... OK
  Applying utils.0002_auto_20180329_2146... OK
  Applying utils.0003_auto_20181112_2150... OK
  Applying utils.0003_auto_20181112_0637... OK
  Applying utils.0004_merge_20181115_1717... OK
  Applying utils.0004_auto_20181208_2202... OK
  Applying utils.0005_merge_20190126_0803... OK
Importing data from JSON (python3 manage.py shell < contenttype.py)...
Importing data from JSON (python3 manage.py loaddata datadump.json)...
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
psycopg2.IntegrityError: null value in column "advertised_prefix_count" violates not-null constraint
DETAIL:  Failing row contains (533, 2001:504:1a::35:120, Jan 3 - Waiting for their confirmation before we enable the sess..., 114, 2, f, null, null, null, , 2019-01-03 18:21:10.423+00, 2019-01-03 18:21:10.423+00, null, f).
```

**Comment by jlixfeld** [comment]
Seems null values are somehow being inserted into `advertised_prefix_count` and `received_prefix_count` but the postgres schema doesn't want null values there.

```
sqlite> select * from peering_internetexchangepeeringsession;
id          ip_address            comment     autonomous_system_id  internet_exchange_id  enabled     bgp_state    advertised_prefix_count  received_prefix_count  password    created                     updated                     last_established_state    
----------  --------------------  ----------  --------------------  --------------------  ----------  -----------  -----------------------  ---------------------  ----------  --------------------------  --------------------------  --------------------------
...
...
533         2001:504:1a::35:120   Jan 3 - Wa  114                   2                     0                                                                                    2019-01-03 18:21:10.423555  2019-01-03 18:21:10.423590                            
534         206.108.35.120        Jan 3 - Wa  114                   2                     0                                                                                    2019-01-03 18:21:10.426526  2019-01-03 18:21:10.426554                            
535         2001:504:1a::35:83    -Disabled   115                   2                     1                                                                                    2019-01-23 19:16:47.086792  2019-01-23 21:30:27.873417                            
536         206.108.35.83         -Disabled   115                   2                     1                                                                                    2019-01-23 19:16:47.090716  2019-01-23 21:30:49.940935                    
```

```
sqlite> update peering_internetexchangepeeringsession set advertised_prefix_count=0, received_prefix_count=0 where id=533;  
sqlite> update peering_internetexchangepeeringsession set advertised_prefix_count=0, received_prefix_count=0 where id=534;
sqlite> update peering_internetexchangepeeringsession set advertised_prefix_count=0, received_prefix_count=0 where id=535;
sqlite> update peering_internetexchangepeeringsession set advertised_prefix_count=0, received_prefix_count=0 where id=536;
```

Those affected peering sessions were added after:

```
root@peeringmanager:/opt/peering-manager.backup# git log -1
commit 5d0ea2b760ca6135b7c5c48e23b0e4748d70a0cb
Author: Guillaume Mazoyer <guillaume@mazoyer.eu>
Date:   Tue Nov 20 16:27:58 2018 +0100

    Log merged configuration, debug help for #71.
root@peeringmanager:/opt/peering-manager.backup# 
```

Migration to postgres was attempted using: 

```
root@peeringmanager:/opt/peering-manager.new# git log -1
commit 7e6ec86ca254ec187986cea0fd9a56dc442fade4
Author: Guillaume Mazoyer <guillaume@mazoyer.eu>
Date:   Sat Jan 26 12:53:22 2019 +0100

    Fix peering sessions import.
root@peeringmanager:/opt/peering-manager.new#
```

**Comment by respawner** [comment]
Will be addressed with the fix for #83.
